c/extras/py_ftrl.cc

```diff
@@ -304,9 +276,10 @@ oobj Ftrl::predict(const PKArgs& args) {
 
   size_t n_features = dtft->get_n_features();
   if (dt_X->ncols != n_features && n_features != 0) {
-    throw ValueError() << "Can only predict on a frame that has "<< n_features
-                       << " column(s), i.e. has the same number of features as "
-                       << "was used for model training";
+    throw ValueError() << "Can only predict on a Frame that has " << n_features
+                       << " column" << (n_features == 1? "" : "s")
```

Comment:
I guess at some point I saw a `plural` function in datatable, but when I wanted to use it here, I couldn't find it. Do I confuse it with something, or there is really something like that?

Reply:
Ah, that explains why I couldn't find it at some point :)